portfolio/src/main/webapp/index.html

```diff
@@ -84,8 +84,8 @@ <h2 id = "cooking" > Cooking </h2>
 		</p>
         <div class="row"> 
         <div class="column">
-			<a target="_blank" href="https://www.bettycrocker.com/recipes/pineapple-upside-down-mini-cakes/0461491a-24a1-4175-a71f-ae43ba0b4db4">
-				<img src="/images/pineapple.jpg" alt="Mini Pineapple Upside Down"> </a>
+			<a target="_blank" href="https://www.allrecipes.com/recipe/18465/gnocchi-i/">
```

Comment:
Nit for future PRs: this part is changing content, which is unrelated to the PR stated intent of changing style.